startup-jars.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -e
+
+tput setaf 4
+cat << 'EOF'
+__     __        _  __         _   _       _        ____  ___  
+\ \   / /__ _ __(_)/ _|_   _  | | | |_   _| |__    / ___|/ _ \ 
+ \ \ / / _ \ '__| | |_| | | | | |_| | | | | '_ \  | |  _| | | |
+  \ V /  __/ |  | |  _| |_| | |  _  | |_| | |_) | | |_| | |_| |
+   \_/ \___|_|  |_|_|  \__, | |_| |_|\__,_|_.__/   \____|\___/
+                       |___/
+                       ...................with jars
+EOF
+tput sgr0
+
+source lib/services.sh
+
+# Generate PKI and config if necessary
+if test ! -d data; then
+  command -v cfssl >/dev/null || brew install cfssl
+  ./generate/hub-dev-pki.sh
+fi
+
+mkdir -p logs
+
+( bin/metadata_server >logs/metadata_server.log 2>&1 & )
+( bin/ocsp_responder >logs/ocsp_responder.log 2>&1 & )
+
+build_service ../verify-hub
+#build_service ../ida-sample-rp
```

Comment:
are these supposed to be commented?

Reply:
no, good spot, they were undone by a later commit :D

Reply:
oh right :)